src/libs/ReportActionsUtils.ts

```diff
@@ -189,7 +189,14 @@ function isActionOfType<T extends ReportActionName[]>(
 ): action is {
     [K in keyof T]: ReportAction<T[K]>;
 }[number] {
-    return actionNames.includes(action?.actionName as T[number]);
+    const actionName = action?.actionName as T[number];
+
+    // This is purely a performance optimization to limit the 'includes()' calls on Hermes
+    if (actionNames.length === 1) {
+        return actionNames[0] === actionName;
+    }
+
+    return actionNames.includes(actionName);
```

Comment:
I'm unfamiliar with the `includes` performance issue on Hermes, but what about having done this instead:

```suggestion
    // This is purely a performance optimization to limit the 'includes()' calls on Hermes
    for (let i = 0; i < actionNames.length; i++) {
        if (actionNames[i] === actionName) {
            return true;
        }
    }
    return false;
```

if the objective is to avoid `includes` wouldn't this help for the case `length === 1` but also for bigger arrays?
Also, should we add some linter rules to avoid `includes`?

Reply:
@aldo-expensify I think I have to step back with this 'includes is slow' statement, my bad! Diving deeper into this to test all the scenarios, please see the above ⬆️ comment.

Reply:
> Not sure if starting an iterator for all these will be for better or for worse, will come back here with numbers so we avoid the guess work.
> My bottom line i that we should default to basic equality checks as ~60% of function calls to isActionOfType are eventually just x === y and this should be instant, though it now takes 6-15ms per call.
Just stripping down these 60% should save ~300ms of compute time for these 50 calls.

I'm not sure what you mean by "starting an iterator" here... are you referring to `for (let i = 0;` as an iterator? or some iterator created within `includes`.

I wouldn't expect the `for` loop with an index to be any slower that what you proposed in the PR, except that it should help for bigger arrays too instead of only the ones with length. Of course this is assuming that `includes` is really slower than using an index to check `actionNames[0] === actionName` (or any other index).

Reply:
@aldo-expensify you're right, the `for (...)` will work here the best. The longest array in use here is 70 items, not that long either. I'll put this to test.

Reply:
@allgandalf I'm currently involved in other initiatives, as this is purely a perf improvement are you ok with holding off for a few days? There's no rush with this change.

Reply:
Just pushed an updated version.